Web/starter_project_vanguard/components/home/BuildBetter.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import buildBetterImage from '../../public/img/home-page/bulid-better-image.jpg'
+const BuildBetter = () => {
+  return (
+    <div className="p-5">
+      <div className="flex flex-col items-center justify-center">
+        
+
+        <div className="flex flex-col items-center justify-center">
+          <h1 className="mx-4 text-4xl font-bold text-center sm:mx-11">
+            Let's Build A Better <span className="block">Tomorrow</span>
+          </h1>
+        </div>
+
+       
+        <div className="pl-10 pr-10 mt-9">
+          <p className="text-center text-gray-700 md:text-lg">
+            A2SV up-skills high-potential university students, connects them
+            with opportunities at top tech companies around the world, and
+            creates digital solutions to urgent problems in their home
+            countries. The program is free for students, making the opportunity
+            available for youth who have the talentbut lack the mans to use it.
+          </p>
+        </div>
+        
+        <div className="p-3 mt-10 ml-4 font-semibold text-white bg-blue-600 border-2 border-blue-500 rounded">
+          <Link href="/">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
```

Comment:
consider using react icons in place of svgs, I think it will make your code cleaner.